### PR TITLE
issue: Wrong Attachment Names

### PR DESCRIPTION
**Pull request by JediKev**
This addresses an issue where FileUploadFields are displaying the old names of existing files with new names.

### CODE REVIEW

File: include/class.forms.php [hunk 1]
```diff
@@ -3896,28 +3896,27 @@ function render($options) {
             function($t) { return strpos($t, '/') !== false; }
         );
         $maxfilesize = ($config['size'] ?: 1048576) / 1048576;
-        $files = $F = array();
+        $files = array();
```
Comment (protich) [inline include/class.forms.php]: unused variable??

**Comment by JediKev** [comment]
Looking at it now, I need to replace it with `$attachments` and possibly replace `$F[] = $a->file;` with `$attachments[] = $a;`... what do you think?

File: include/class.forms.php (hunk 1 above)
Comment (JediKev) [reply include/class.forms.php]: Looking at it now, I need to replace it with `$attachments`. what do you think?

Comment (protich) [reply include/class.forms.php]: Nothing to think about - delete it - it's not being used, right?

**Pull request merged**